packages/eas-cli/src/commands/workflow/run.ts

```diff
@@ -0,0 +1,99 @@
+import chalk from 'chalk';
+import fs from 'node:fs';
+import * as path from 'node:path';
+
+import { getWorkflowRunUrl } from '../../build/utils/url';
+import EasCommand from '../../commandUtils/EasCommand';
+import { WorkflowProjectSourceType } from '../../graphql/generated';
+import { WorkflowRunMutation } from '../../graphql/mutations/WorkflowRunMutation';
+import Log, { link } from '../../log';
+import { getOwnerAccountForProjectIdAsync } from '../../project/projectUtils';
+import { uploadAccountScopedEasJsonAsync } from '../../project/uploadAccountScopedEasJsonAsync';
+import { uploadAccountScopedProjectSourceAsync } from '../../project/uploadAccountScopedProjectSourceAsync';
+
+export default class WorkflowRun extends EasCommand {
+  static override description = 'Run an EAS workflow';
+  static override aliases = ['workflow:run'];
+  static override usage = [chalk`workflow:run {dim [options]}`];
+
+  // TODO(@sjchmiela): Keep command hidden until workflows are live
+  static override hidden = true;
+  static override state = 'beta';
+
+  static override args = [{ name: 'file', description: 'Path to the workflow file to run' }];
+
+  static override contextDefinition = {
+    ...this.ContextOptions.DynamicProjectConfig,
+    ...this.ContextOptions.ProjectDir,
+    ...this.ContextOptions.Vcs,
+    ...this.ContextOptions.LoggedIn,
+  };
+
+  async runAsync(): Promise<void> {
+    const { flags, args } = await this.parse(WorkflowRun);
+
+    Log.warn('Workflows are in beta and subject to breaking changes.');
+
+    const {
+      getDynamicPrivateProjectConfigAsync,
+      loggedIn: { graphqlClient },
+      vcsClient,
+      projectDir,
+    } = await this.getContextAsync(WorkflowRun, {
+      nonInteractive: flags['non-interactive'],
```

Comment:
Wait, how does it even work? 😮 

It doesn't seem that this command defines any flags, so `flags['nonInteractive']` will always be `undefined`? Why it isn't a TypeError 🤯?

Reply:
It seems like we get type hints for existing flags defined in the `flags` field, but it doesn't warn/throw an error if you access a flag that doesn't exist. It's weird. It seems dangerous and error-prone

Reply:
Applied your suggestion adding the flag.